_maps/configs/syndicate_lugol.json

```diff
@@ -24,6 +24,10 @@
 			"outfit": "/datum/outfit/job/bartender/syndicate",
 			"slots": 1
 		},
+		"Resource Haulier": {
```

Comment:
I'm gonna donk you for this and the Potation whatever. 

Weird Job Names are bad for people who don't already know what the job is, and this is much better than potation specialist but you should change both of them anyays